tox.ini

```diff
@@ -1,26 +1,72 @@
 [tox]
 envlist =
     py35,
+    py35-deps,
     py36,
+    py36-deps,
     py37,
+    py37-deps,
     py38,
+    py38-deps,
     lint-py36
     coverage-report
 
 [testenv]
 coverage_run = {env:WIN_COV:coverage run}
+use_develop = True
 commands =
-    {envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} install test alltests
-    {[testenv]coverage_run} {envdir}/bin/alltests {posargs:-vc}
-skip_install = true
+    {[testenv]coverage_run} -m zope.testrunner \
```

Comment:
This will no longer work on appveyor on `master` it was used to disable coverage on windows.